app/(main)/register/page.test.tsx

```diff
@@ -170,4 +173,31 @@ describe('Register', () => {
 
     expect(darkModeSection).toHaveClass('dark:bg-gradient-to-b');
   });
+
+  test('Should show loadingspinner in submit button when clicked', async () => {
+    jest.mock('../../../api/apiFunctions', () => ({
+      registerAccount: jest.fn(() => {
+        setTimeout(() => {
+          console.log('register account done running...');
+        }, 100000);
+      }),
+    }));
+
+    fireEvent.change(screen.getByTestId('email'), {
+      target: { value: 'test5@test.com' },
+    });
+    fireEvent.change(screen.getByTestId('password'), {
+      target: { value: 'password12345' },
+    });
+    fireEvent.change(screen.getByTestId('confirm-password'), {
+      target: { value: 'password12345' },
+    });
+
+    fireEvent.click(continueButton);
+
+    waitFor(async () => {
```

Comment:
You should not need to set the async here again. Follow how Mai did it in this test.
https://github.com/LetsGetTechnical/gridiron-survivor/pull/454/files#diff-4c664975b58eb9a2b8fd8520303503caa5c63ea6e1c3f47e2682744b30c2623eR45